storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/ChangesFilter.java

```diff
@@ -16,28 +17,62 @@
  * <p>
  * Hides RxJava from ClassLoader via separate class.
  */
-final class ChangesFilter implements Func1<Changes, Boolean> {
+public final class ChangesFilter implements Func1<Changes, Boolean> {
```

Comment:
No, because it used in `com.pushtorefresh.storio.sqlite.impl` for `DefaultStorIOSQLite#observeChangesInTables` and `com.pushtorefresh.storio.sqlite.operations.execute` for `asRxObservable`